### PR TITLE
Add healthcheck.sh.

### PR DESCRIPTION
**Pull request by leszko**
Comments to PR:
- this version of healthcheck.sh uses the /hazelcast/health endpoint and parses the result, imo the endpoint should be extended to provide more granularity, but until discussed, this version works fine
- script follows the conventions used in cluster.sh, so that it would be possible to wrap all scripts with one master command, like, for example, `hazelcast healthcheck ...`

### CODE REVIEW

**Comment by mesutcelik** [comment]
```
bin$ healthcheck.sh 
No operation is defined, running script with default operation: 'all'.
No port is defined, running script with default port: '5701'.
No specific ip address is defined, running script with default ip: '127.0.0.1'.
Error while checking health of hazelcast cluster on ip 127.0.0.1 on port 5701.
Please check that cluster is running and that health check is enabled (property set to true: 'hazelcast.http.healthcheck.enabled' or 'hazelcast.rest.enabled').
```
I feel this is too crowded. We might remove first 3 lines becasue ip and port is already mentioned in 4th line.  And not sure we need to say the first line...

**Comment by leszko** [comment]
Yes, it's crowded, but it also keeps the same convention as cluster.sh, which have some value IMO. I would go into the direction of keeping it similar and adding later to one CLI tool. wdyt?

**Comment by mesutcelik** [comment]
ok fine but then maybe adding those features into `cluster.sh` would make more sense...

File: hazelcast/src/main/resources/healthcheck.sh [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    echo "Please check that cluster is running and that health check is enabled (property set to true: 'hazelcast.http.healthcheck.enabled' or 'hazelcast.rest.enabled')."
+    exit 1
+fi
+
+if [ "$OPERATION" = "all" ]; then
+    echo "${HTTP_BODY}"
+    exit 0
+fi
+
+RESULT=$(echo $HTTP_BODY | awk "{match(\$0,  /Hazelcast\:\:${KEYWORD}\=([A-Z_0-9]*)/, m)}END{print m[1]}")
```
Comment (mesutcelik) [inline hazelcast/src/main/resources/healthcheck.sh]: Can you please also check this one as we discussed internally?

```
bin$ healthcheck.sh -a hazelcast-service-hazelcast.192.168.64.2.nip.io -p 80 -o cluster-size
awk: syntax error at source line 1
 context is
    {match($0,  >>>  /Hazelcast\:\:ClusterSize\=([A-Z_0-9]*)/, <<< 
awk: illegal statement at source line 1
awk: illegal statement at source line 1
```

```
bin$ awk --version
awk version 20070501
```

**Comment by leszko** [comment]
No, the cluster.sh has a completely different purpose and usage, so I'd keep it separate. Anyway, I removed the messages about the default parameters. I see you have a strong opinion on that and I'm somewhere on a fence (I don't like them, but I like the consistency with cluster.sh).

File: hazelcast/src/main/resources/healthcheck.sh (hunk 1 above)
Comment (leszko) [reply hazelcast/src/main/resources/healthcheck.sh]: The issue was related to the fact that on Mac by default a very old version of 'awk' is installed (year 2007!). I changed 'awk' to 'grep' + 'sed' and tested also on Mac. Should work fine now.

**Comment by mesutcelik** [comment]
@hasancelik Can you please check if that is included in our zip/tar.gz artifacts without changing release script. 
I also think that needs to be included in reference manual. cc: @Serdaro

**Pull request merged**